### DISCUSSION TITLE
[react-markdown] Does not wrap in the <em/> tag, if `emphasis` is set in `allowedElements`.

### DISCUSSION THREAD

**Discussion by iliatapazukk** [remarkjs/remark#892] [Q&A] [answered]
Subj. It's a bug?

```js
const testText = '_qweqwe_'
<ReactMarkdown
          allowedElements={['emphasis', 'text', 'strong', 'delete', 'link']}
          unwrapDisallowed={true}
          children={testText}
          skipHtml={true}
        />
```

**Comment by wooorm** [comment, marked as answer]
A bug in your code, yes ;)
See: https://github.com/remarkjs/react-markdown/blob/main/changelog.md#replace-allownode-allowedtypes-and-disallowedtypes, https://github.com/remarkjs/react-markdown#appendix-b-components

**Reply by iliatapazukk** [reply]
I'm sorry, but according to the documentation I use `allowedElements`. Or should a different exception be used for `emphasis`? Where is a bug?

**Reply by wooorm** [reply]
`emphasis` -> `em`

**Reply by iliatapazukk** [reply]
Thanks!

**Reply by ChristianMurphy** [reply]
@iliatapazukk it looks like you updated from react-markdown <6 to 6 or newer.
As part of that update some migration of options is needed https://github.com/remarkjs/react-markdown/blob/main/changelog.md#replace-allownode-allowedtypes-and-disallowedtypes
Note that when updating `allowNode`, the option is not just renamed, it accepts different input, it uses HTML elements not Markdown nodes.
The example you provided uses Markdown nodes, those need to be updated to HTML, there is conversion table in https://github.com/remarkjs/react-markdown/blob/main/changelog.md#change-renderers-to-components (unfurl "Show conversion table") which shows how those map
